languages/en.ini

```diff
@@ -1515,6 +1517,7 @@ The system is currently unavailable due to system maintenance = "The system is c
 Theme = "Theme"
 Thesis = "Thesis"
 This email was sent from = "This email was sent from"
+This feature = "This feature"
```

Comment:
For context, maybe you could use as a key here, `default_service_description` instead of `This feature`.